### ISSUE TITLE
`cmake` errors when building desktop on M1 mac

### ISSUE THREAD

**Issue by alexgurr** [BelledonneCommunications/linphone-desktop#501]
**Describe the bug**
When trying to build the desktop app locally and following the build instructions, the build fails (see below under logs).

Command that fails -- `cmake --build . --target install --parallel 10 --config RelWithDebInfo` with the following errors:

```bash
make[11]: *** [simd/CMakeFiles/simd.dir/arm64/jsimd_neon.S.o] Error 1
make[10]: *** [simd/CMakeFiles/simd.dir/all] Error 2
make[9]: *** [all] Error 2
make[8]: *** [/Users/alexgurr/Documents/cloudwave/apps/linphone-desktop/build/WORK/WORK/desktop/Stamp/EP_turbojpeg/EP_turbojpeg-build] Error 2
make[7]: *** [CMakeFiles/EP_turbojpeg.dir/all] Error 2
make[6]: *** [all] Error 2
make[5]: *** [sdk-prefix/src/sdk-stamp/sdk-build] Error 2
make[4]: *** [CMakeFiles/sdk.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [WORK/stamp/sdk-build] Error 2
make[1]: *** [CMakeFiles/sdk.dir/all] Error 2
make: *** [all] Error 2
```

If this is relevant more to the SDK, happy to move to that repository.

**To Reproduce**
The build steps in the README file on an M1 mac.

**Expected behavior**
It builds correctly.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: 11.2.3
 - SDK + desktop versions: latest

**SDK logs**
https://paste.ee/p/hF1vq

**Comment by julonexus** [comment]
Hi,
Linphone doesn't support a build on M1. But you can test this branch : https://gitlab.linphone.org/BC/public/linphone-sdk/-/tree/feature/apple_arm

**Comment by alexgurr** [comment]
How do I specify this single branch for the nested SDK within desktop, when I am checking out desktop directly (`git clone https://gitlab.linphone.org/BC/public/linphone-desktop.git --recursive`)

**Comment by julonexus** [comment]
Hi
I guess you are trying to build with Xcode generator. This generator doesn't work yet. This is still to be resolved but it should be fine with a Unix Makefiles.

**Comment by amitiyer** [comment]
Any update on the M1 Build issues ? I still cannot build Linphone-Desktop on M1 Mac

**Comment by julonexus** [comment]
Only Qt6 support M1. But the application doesn't support it. So there is no solution.

**Comment by julonexus** [comment]
In fact Qt will support officially the M1 for 5.15.9.
Some have built the application on M1 with 5.15.6 with a fix that was done on SDK 5.2 (you have to use the master of Linphone Desktop from now).
I you have still issues, please open another post with all the build log.
